concent_api/core/message_handlers.py

```diff
@@ -1345,19 +1365,13 @@ def handle_send_subtask_results_verify(
                 report_computed_task=report_computed_task,
                 subtask_results_rejected=subtask_results_rejected,
             )
-
         else:
-            if subtask.state_enum in [
-                Subtask.SubtaskState.VERIFICATION_FILE_TRANSFER,
-                Subtask.SubtaskState.ADDITIONAL_VERIFICATION,
-            ]:
-                return message.concents.ServiceRefused(
-                    reason=message.concents.ServiceRefused.REASON.DuplicateRequest,
-                )
             if subtask.state_enum in [
                 Subtask.SubtaskState.ACCEPTED,
                 Subtask.SubtaskState.FAILED,
             ]:
+                claim_against_requestor.delete()
+                claim_against_provider.delete()
```

Comment:
Please use `discard_claim()`.

Reply:
We need to discuss, isn't it mentioned in blueprint that Bankster only removes `DepositClaim` objects after they're successfully paid for?

Reply:
See my response for the other comment: https://github.com/golemfactory/concent/pull/922#discussion_r233003576

It looks like I did not write that clearly enough. It's not that no Bankster function can touch these objects. I just meant that Bankster should not do this automatically. It should not decide on its own to remove them as a part of another operation. But if the view tells it explicitly to remove them (by running `discard_claim()`) it's fine to do it.

Reply:
:+1: done